.ci/update-github-io.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: update-github-io.sh <version>"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+CURRENT_VERSION=$(xmlstarlet sel -N pom=http://maven.apache.org/POM/4.0.0 \
+                             -t -m pom:project -v pom:version pom.xml | sed "s/-SNAPSHOT//")
+echo CURRENT_VERSION="$CURRENT_VERSION"
+
+if [ "$TARGET_VERSION" != "$CURRENT_VERSION" ]; then
```

Comment:
This action happens after version bump is done.
Remove this validation

Reply:
Changes Done